cmd/check-gh-automation/README.md

```diff
@@ -1,5 +1,6 @@
 # Check GH Automation
-A tool to check that our bots (`openshift-merge-robot` and `openshift-ci-robot`) have access to repositories that have CI configured.
+A tool to check that our bots (`openshift-merge-robot`, `openshift-ci-robot`, and `openshift-cherrypic
+k-robot`) have access to repositories that have CI configured.
```

Comment:
I realize that when viewing the file on the browser this makes no difference, but there is a hard return in the middle of a word here.